modules/io/common/src/main/java/org/locationtech/jts/io/geojson/OrientationTransformer.java

```diff
@@ -0,0 +1,85 @@
+package org.locationtech.jts.io.geojson;
+
+import org.locationtech.jts.algorithm.Orientation;
+import org.locationtech.jts.geom.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OrientationTransformer {
```

Comment:
Add class Javadoc.  Can be something simple like

> Utilities to modify the ring orientation of polygonal geometries.

Reply:
Took your recommendation verbatim